charts/hedera-block-node/README.md

```diff
@@ -11,23 +11,35 @@ minikube delete && minikube start --kubernetes-version=v1.23.0 --memory=8g --boo
 ```
 
 Set environment variables that will be used for the remainder of the document:
+Replacing the values with the appropriate values for your environment: bn1 is short for block-node-1, but you can name you release as you wish. And use the version that you want to install.
 ```bash
 export RELEASE="bn1"
```

Comment:
The name `bn1` is not great, even as a "please replace this" default.  I would suggest something more useful, such as `Hedera Block Node`.

Reply:
Hmmm, `bn1` is still not great, `bn` is OK, and `blkNod` would be better.
Adding numbers tends to create issues for managing clusters more than help.

Reply:
Yes, Agreed, will update it to `blkNod` since even when is only an example, most people might just use it as is 😓

Reply:
✅ Done!